### ISSUE TITLE
mock mode support

### ISSUE THREAD

**Issue by ianzone** [nylas/nylas-nodejs#462]
be able to create a mock nylas instance that can return mock data when running unit tests.

**Comment by pgsnimble** [comment]
Good idea, it would make easier to integration tests. Another approach that you could follow is to use nock in order to mock Nylas requests. It works very well.

But I agree that having this already implemented in the sdk would help a lot
